app/assets/stylesheets/scoreboard.scss

```
@@ -21,3 +21,13 @@
     }
   }
 }
+
+td.last-updated {
+  animation: 1s linear 0s 5 blinkCell
```

```suggestion
  animation: 0.75s ease-in-out 0s 3 blinkCell
```
the `ease-in-out` is lesser to the 0.75s/3 reps change

---

So the scoreboard is literally a TV that's just posted and people walk by and glance at it. I think shortening it too much would effectively counteract adding it all as no one will ever see it. 

Sure for `ease-in-out`, i don't really see a difference